junebug/__init__.py

```diff
@@ -2,6 +2,26 @@
 Junebug is a system for managing text messaging transports via a RESTful HTTP
 interface.
 '''
+import os
+
+# Allows us to select which Twisted reactor to use. Must be done before any
+# Twisted import calls.
+r = os.environ.get('JUNEBUG_REACTOR')
+if r == "SELECT":
+    from twisted.internet import selectreactor as r
+elif r == "POLL":
+    from twisted.internet import pollreactor as r
+elif r == "KQUEUE":
+    from twisted.internet import kqreactor as r
+elif r == "WFMO":
+    from twisted.internet import win32eventreactor as r
+elif r == "IOCP":
+    from twisted.internet import iocpreactor as r
+elif r == "EPOLL":
+    from twisted.internet import epollreactor as r
+if r is not None:
```

Comment:
If someone sets `JUNEBUG_REACTOR` to `silly-typo`, this will attempt to call `.install()` on a string, which won't be good. Perhaps we should add a final `eilf`:
```python
elif r is not None:
    raise RuntimeError("Unsupported JUNEBUG_REACTOR setting %r" % (r,))
```
Might also be worthwhile allowing explicitly setting the string `DEFAULT` to get the default reactor (Twisted has `twisted.internet.default.install` if that helps).